cli_tools_e2e_test/common/utils/utils.go

```diff
@@ -131,9 +131,13 @@ func runCliTool(logger *log.Logger, testCase *junitxml.TestCase, cmdString strin
 }
 
 // RunTestCommand runs given test command
-func RunTestCommand(cmd string, args []string, logger *log.Logger, testCase *junitxml.TestCase) bool {
+func RunTestCommand(cmd string, args []string, logger *log.Logger, testCase *junitxml.TestCase, allowError bool) bool {
```

Comment:
`allowError` is a big vague. Does this mean the test case is asserting that an error should occur? If it means "maybe an error can occur, and that's okay" -- what's an example of this?

Reply:
In the past, we run a command and always expect success (return value=0).
Now, my new test needs to run a command and I know it will fail - which is what I am going to test, so it shouldn't mark the test case fail.
Maybe let me make a new function called "RunTestCommandIgnoringError".